src/corelibs/U2Core/src/globals/GUrl.cpp

```diff
@@ -143,6 +147,26 @@ bool GUrl::operator !=(const GUrl& url) const {
     return !(*this == url);
 }
 
+// The function converts url string to multibyte form
+// default code page is CP_THREAD_ACP
+const char* GUrl::getURLStringAnsi(int codePage) const {
+#ifdef Q_OS_WIN
+    std::wstring wPath = getURLString().toStdWString();
+    codePage = codePage < 0 ? CP_THREAD_ACP : codePage;
+
+    DWORD buffSize = WideCharToMultiByte(codePage, 0, wPath.c_str(), -1, NULL, 0, NULL, NULL);
+    if (!buffSize)
+        return NULL;
```

Comment:
Our standard is c++11 now. Try to use nullptr instead of NULL. It is not critical, of course.

Reply:
done